src/test/resources/shellFindFilesTest/test_query.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE testdb;
```

Comment:
Let's call the folder `HiveRunnerExtesionTest` so it matches the class name exactly. I see we have some data folders here which do that and some which don't. I think we should change the ones that don't to be consistent. Feel free to do that in this PR or a separate one (or just create an issue for someone else to do :) )

Reply:
I'll do it in a separate PR 👍